src/apps/statements/expressPresenter/postStatements/storeWithAttachments.ts

```diff
@@ -17,10 +18,12 @@ export default async ({ config, req, res }: Options) => {
   const client = await getClient(config, defaultTo(req.header('Authorization'), ''));
 
   validateStatementProcessingPriority(req.query.priority as string | undefined);
+  validateStatementBypassQueues(req.query.bypassQueues as string[] | undefined);
```

Comment:
@lucuilearningpool , it can't be a `string[]`. All values of `req.query` whether `undefined` or a simple `string`

Reply:
Of course, thanks @crazy-grizzly